site/ja/tutorials/quickstart/beginner.ipynb

```diff
@@ -207,19 +203,200 @@
       },
       "outputs": [],
       "source": [
-        "model.fit(x_train, y_train, epochs=5)\n",
+        "predictions = model(x_train[:1]).numpy()\n",
+        "predictionspredictions = model(x_train[:1]).numpy()\n",
```

Comment:
[suggestion] This line does not exist in https://raw.githubusercontent.com/tensorflow/docs/master/site/en/tutorials/quickstart/beginner.ipynb

Reply:
Same as above, I'll fix this issue by applying the `nb_code_sync.py` tool.